### PR TITLE
[COMMONS-1] Created commons project with credential and shell executi…

### LINKED ISSUE #1: Extract common utilities out of individual projects for ease of repeated use
Noted candidates:
- Credential helpers
- Shell execution helpers

### PR DESCRIPTION
**Pull request by peter-mcclonski**
…on utilities

### CODE REVIEW

**Comment by peter-mcclonski** [comment]
@TechnologyBrewery/technology-brewery-maintainers 
_Very_ WIP.  Interested in opinions on the general structure/approach.  Obviously needs tests.

Example (also WIP) usages: 
- https://github.com/TechnologyBrewery/habushu/pull/51
- https://github.com/TechnologyBrewery/orphedomos/pull/23

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        <artifactId>cucumber-bom</artifactId>
+        <version>7.12.0</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
+    </dependencies>
+  </dependencyManagement>
+
+  <build>
+    <pluginManagement><!-- lock down plugins versions to avoid using Maven defaults (may be moved to parent pom) -->
```
Comment (d-ryan-ashcraft) [inline pom.xml:37]: A: I think most of these are already there

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+  <packaging>pom</packaging>
+  <modules>
+    <module>shell</module>
+    <module>credential</module>
+  </modules>
+
+  <parent>
+    <groupId>org.technologybrewery</groupId>
+    <artifactId>parent</artifactId>
+    <version>1</version>
```
Comment (d-ryan-ashcraft) [inline pom.xml:15]: S: should be version 6, I think this will pull in most if not all of the plugin versions from the other comment

**Review by d-ryan-ashcraft — COMMENTED** [review]
Direction looks good to me

**Review by d-ryan-ashcraft — APPROVED** [review]
This is WIP, but need the basic structure to add in some other items.  Since nobody will be using the credential and shell modules with that, we'll push these in and then modify them in a subsequent PR.

**Pull request merged**